docs/db/changelog/changesets/user-data-store/1.0.x/20231003-add-tag-1.0.0.xml

```diff
@@ -3,7 +3,7 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
 
-    <changeSet id="1" logicalFilePath="user-data-store/1.0.x/20231003-add-tag-1.0.0" author="Lubos Racansky">
+    <changeSet id="1" logicalFilePath="user-data-store/1.0.x/20231003-add-tag-1.0.0.xml" author="Lubos Racansky">
```

Comment:
😬 I guess that it affects the checksum, doesn't it?

Reply:
The checksum is calculated based on the changeSet's ID, author, and the content of the changeSet itself, so it should be fine.

Reply:
So I should revert the change, right? Or do you propose any other solution?

Reply:
yes please revert the change. The form with .xml is correct but the change will case to have two tags in the databasechangelog.